packages/dds/tree/src/test/shared-tree/fuzz/fuzzUtils.ts

```diff
@@ -24,38 +24,77 @@ import {
 } from "../../../core/index.js";
 import { SchemaBuilder, leaf } from "../../../domains/index.js";
 import {
+	Any,
 	FieldKinds,
 	FlexFieldSchema,
 	FlexTreeObjectNodeTyped,
+	LeafNodeSchema,
+	SchemaLibrary,
 	intoStoredSchema,
 } from "../../../feature-libraries/index.js";
 import { ITreeCheckout, SharedTree } from "../../../shared-tree/index.js";
 import { expectEqualPaths } from "../../utils.js";
 
 const builder = new SchemaBuilder({ scope: "tree2fuzz", libraries: [leaf.library] });
-export const fuzzNode = builder.objectRecursive("node", {
-	requiredChild: FlexFieldSchema.createUnsafe(FieldKinds.required, [
-		() => fuzzNode,
-		...leaf.primitives,
-	]),
-	optionalChild: FlexFieldSchema.createUnsafe(FieldKinds.optional, [
-		() => fuzzNode,
-		...leaf.primitives,
-	]),
-	sequenceChildren: FlexFieldSchema.createUnsafe(FieldKinds.sequence, [
-		() => fuzzNode,
-		...leaf.primitives,
-	]),
+
+// We write any here, but for purpose of fuzz test it's fuzz node or primitive
```

Comment:
I'd elaborate on this comment.

"We use any here to give compile-time flexibility, but during a fuzz test's runtime, different trees will have different views over the currently allowed schema. This extremely permissive schema is a valid superset over all possible schema and is a reasonable type to use at compile time, but generators/reducers working with trees over the course of a fuzz test need to be careful to appropriately narrow their edits to be valid for the tree's current schema at runtime."

Reply:
It's probably worth putting an explanation of what model the fuzz tests use w.lr.t schema in operationTypes.ts.

Reply:
also, convert this to a doc comment so that people see it on mouseover of `fuzzNode` :)

Reply:
Updated comment to docstring, and elaborated further with suggestions.